### PR TITLE
Update ug sort command

### LINKED ISSUE #138: Update UG for sort
As a financial advisor, I need an updated User Guide for the sort command, so that I can refer to it when I face some issues.

### PR DESCRIPTION
**Pull request by syamfarh**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -408,61 +408,72 @@ For undoing a clear command:
...
-* `appt/` sort by Appointment Date parameter
+Acceptable prefix tag:
+* `n/` sort by Name prefix
+* `appt/` sort by Appointment Date prefix
 
-![edit format](images/sort-UG/sortformat.png)
+![sort format](images/sort-UG/sortformat.png)
 
 Precise expected output on success:
 * Successful addition message. 'X person listed!', where X are the number of clients in the address book.
```
Comment (bhnuka) [inline docs/UserGuide.md]: Should be "X is the number of clients..." (not are)

File: docs/UserGuide.md [hunk 2]
```diff
@@ -408,61 +408,72 @@ For undoing a clear command:
...
 * The sorted entry is displayed in the address book GUI.
 
-![edit format](images/sort-UG/sortresult.png)
+![sort format](images/sort-UG/sortresult.png)
 
 Precise expected outputs on failure:
-* If a required parameter is missing, an error message should indicate the invalid format. 
-* If a parameter provided is invalid (e.g., e/), an error message should indicate the invalid format.
+<br> If a required parameter is missing, an error message should indicate the invalid format. 
+<br> If a parameter provided is invalid (e.g., e/), an error message should indicate the invalid format.
```
Comment (bhnuka) [inline docs/UserGuide.md]: should combine these into one line (i.e If a require parmeter is missing or invalid, an error message should...)
alternatively, can have two screenshots and messages for each. but thats a bit repetitive so best to stick to combining them.  would be cleaner!

**Review by bhnuka — COMMENTED** [review]
Tiny nits. Should be easy to fix.

File: docs/UserGuide.md (hunk 1 above)
Comment (syamfarh) [reply docs/UserGuide.md]: fixed! thanks!

File: docs/UserGuide.md (hunk 2 above)
Comment (syamfarh) [reply docs/UserGuide.md]: noted fixed! but it might look very long

File: docs/UserGuide.md [hunk 3]
```diff
@@ -408,61 +408,71 @@ For undoing a clear command:
...
+* Successful addition message. 'X person listed!', where X is the number of clients in the address book.
 * The sorted entry is displayed in the address book GUI.
 
-![edit format](images/sort-UG/sortresult.png)
+![sort format](images/sort-UG/sortresult.png)
 
 Precise expected outputs on failure:
-* If a required parameter is missing, an error message should indicate the invalid format. 
-* If a parameter provided is invalid (e.g., e/), an error message should indicate the invalid format.
+<br> If a required parameter is missing, an error message should indicate the invalid format. If a parameter provided is invalid (e.g., e/), an error message should indicate the invalid format.
```
Comment (bhnuka) [inline docs/UserGuide.md]: no syam as in, you dont have to repeat "an error message should indicate the invalid format" twice haha

change the line to exactly this

<br> If either a required parameter is missing or a parameter provided is invalid (e.g., e/), an error message should indicate the invalid format.

Comment (syamfarh) [reply docs/UserGuide.md]: OH HAHAHAH FIXED

**Comment by bhnuka** [comment]
LGTM!

**Pull request merged**